<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoTravelSetup</class>
 <widget class="QMainWindow" name="AutoTravelSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>其他参量监测</string>
    </property>
    <widget class="QCheckBox" name="batteryCBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>电池温度</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cpuTempCBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>106</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU温度</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="batteryLineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cpuTempLineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>106</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="batteryLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>警戒值(M)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="cpuTempLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>106</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>警戒值(%)：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="wifiCBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>133</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wifi流量</string>
     </property>
    </widget>
    <widget class="QLabel" name="wifiLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>133</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>警戒值（M）：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="wifiLineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>133</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="mobileLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>警戒值（M）：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="mobileCBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mobile流量</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mobileLineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="memCBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>28</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>内存</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cpuCBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>54</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="memLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>28</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>警戒值(M)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="cpuLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>54</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>警戒值(%)：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="memLineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>28</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cpuLineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>54</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>日志关键字过滤</string>
    </property>
    <widget class="QLineEdit" name="logKeyWordLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> ANR ;FATAL</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>统计点</string>
    </property>
    <widget class="QRadioButton" name="debugRBtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>debug</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="releaseRBtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>release</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>APP版本：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IPLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="portLineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTreeWidget" name="xmlTreeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>331</width>
       <height>181</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="selectXmlBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>331</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>点击选择XML文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>点击间隔(ms)</string>
    </property>
    <widget class="QLineEdit" name="timeLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="statCBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>统计点测试</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
